chroma-manager.conf.template

```diff
@@ -15,6 +15,8 @@ map $uri $cache_control {
 }
 
 server {
+    root /var/lib/chroma;
```

Comment:
Have you made sure everything still works? changing the root of the whole server is a big change.

Reply:
The current nginx root is located at `/usr/share/nginx/html` so in theory nothing should be affected on our end. With that said, I ran the stratagem integration test and also run stratagem with both the report and purge options and both were successful. With that said, if you feel uncomfortable I can revert this line, I will just need to update the block below such that that paths are based on /usr/share/nginx/html instead of /var/lib/chroma.

Reply:
Ok. If you think this looks good then let's go with it.